tests/LogHandlerTest.php

```diff
@@ -66,6 +61,43 @@ public function rawNotification(callable $fn): array
         $this->assertArrayHasKey('time', $reporter->notification['server']);
         $this->assertEquals('production', $reporter->notification['server']['environment_name']);
 
+        $this->assertEquals([
+            'class' => 'ERROR Log',
+            'message' => 'Test log message',
+            'tags' => [
+                'log',
+                'test-logger.ERROR',
+            ],
+        ], array_only($reporter->notification['error'], ['class', 'tags', 'message']));
+
+        $this->assertFalse(empty($reporter->notification['error']['fingerprint']));
+    }
+
+    /** @test */
+    public function it_ignores_logs_below_its_minimum_level()
+    {
+        $reporter = new class extends Honeybadger {
+            public $notification;
+
+            public function rawNotification(callable $fn): array
+            {
+                $this->notification = $fn($this->config);
+
+                return [];
+            }
+        };
+
+        $logger = new Logger('test-logger');
+        $logger->pushHandler(new LogHandler($reporter, Logger::ERROR));
+
+        $logger->info('Test log message', ['some' => 'data']);
+
+        $this->assertEquals(null, $reporter->notification);
+
+        $logger = new Logger('test-logger');
+        $logger->pushHandler(new LogHandler($reporter, Logger::INFO));
+        $logger->info('Test log message', ['some' => 'data']);
```

Comment:
Looks like this block of code is out of context? Considering the name of the test "it ignores logs below its minimum level".

Reply:
In the first example, the minimum level is ERROR, and we test that it does not write the info log,. In this example, the minimum level is changed to INFO, and we test that it writes the info log.